app/src/main/java/lt/vilnius/tvarkau/utils/AnalyticsUtil.kt

```diff
@@ -0,0 +1,35 @@
+package lt.vilnius.tvarkau.utils
+
+import android.app.Activity
+import android.content.Context
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import com.google.firebase.analytics.FirebaseAnalytics
+import lt.vilnius.tvarkau.entity.Problem
+
+object AnalyticsUtil {
```

Comment:
It depends on our testing approaches, what do we want to test, where do we want to move with tests, how are we planning to develop the app. Let's move this discussion to https://github.com/vilnius/tvarkau-vilniu/pull/102